dashbuilder/dashbuilder-shared/dashbuilder-services-api/src/main/java/org/dashbuilder/transfer/DataTransferServices.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dashbuilder.transfer;
+
+import java.util.List;
+
+import org.jboss.errai.bus.server.annotations.Remote;
+
+@Remote
+public interface DataTransferServices {
+
+    public static final String FILE_PATH = "dashbuilder-data-transfer";
```

Comment:
I agree
However, I need these constants in client classes (DataTransferScreen.java)
What do you suggest ?
If I use Caller<> in client class, it will be 2 AJAX calls under the hood just to get 2 constants, and since it's async, it cannot be returned from java function.
Another option is to use hardcoded string on client class.
Another options is to create a concrete java class just to hold these constants.
I don't see any good way to do it.

Reply:
Oh, I see. It seems there's no a great way to do it so let's leave it as is.